### PR TITLE
[cling-cpt] Split long lines to get rid of E501 error [skip-ci]

### PR DESCRIPTION
**Pull request by saisoma123**
# This Pull request: Makes the code more flake8 compliant.

## Checklist:

- [X] tested changes locally
- [NA] updated the docs (if necessary)

This PR fixes issue #406 mentioned in (https://github.com/root-project/cling/issues/406)

### CODE REVIEW

**Comment by saisoma123** [comment]
Will make more pull requests to fully get rid of the E501 error

File: interpreter/cling/tools/packaging/cpt.py [hunk 1]
```diff
@@ -1923,7 +1923,8 @@ def custom_input(prompt, always_yes=False):
             Do you want to continue? [yes/no]: ''', args['y']).lower()
         if choice in yes:
             subprocess.call(
-                "sudo {0} -m pip install distro".format(sys.executable), shell=True
+                "sudo {0} -m pip" +
+                " install distro".format(sys.executable), shell=True
```
Comment (jalopezg-r00t) [inline interpreter/cling/tools/packaging/cpt.py:1927]: I'd suggest to split at the `,` to avoid breaking the string literal (same below, where it applies).

File: interpreter/cling/tools/packaging/cpt.py [hunk 2]
```diff
@@ -2283,15 +2300,16 @@ def custom_input(prompt, always_yes=False):
...
     args["with_llvm_binary"] = True
 
-    if args["with_binary_llvm"]:
-        download_llvm_binary()
-        compile = compile_for_binary
-        install_prefix = install_prefix_for_binary
+    if args["with_llvm_binary"]:
+        #download_llvm_binary()
+        #compile = compile_for_binary
+        #install_prefix = install_prefix_for_binary
```
Comment (jalopezg-r00t) [inline interpreter/cling/tools/packaging/cpt.py:2312]: Why this additional change?  It's not part of the PR description.

**Review by vgvassilev — APPROVED** [review]
LGTM! I am surprised they only report the issues but to not provide an automatic generate a fix for these coding violations.

**Comment by vgvassilev** [comment]
@saisoma123, can you rebase this PR?

**Comment by saisoma123** [comment]
@vgvassilev Yeah I think there is a way for automatic linting I'll do that instead. I'll also rebase the PR right now

**Comment by saisoma123** [comment]
Yeah just rebased the PR
